jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -373,7 +373,11 @@
             # Get golang into our PATH so we can run e2e.go
             export PATH=${{PATH}}:/usr/local/go/bin
 
-            timeout -k 15m 55m ./hack/jenkins/e2e-runner.sh && rc=$? || rc=$?
+            if [[ -e "./hack/jenkins/.use_dockerized_e2e" ]]; then
```

Comment:
I'm not executing this file - just using it as a marker to trigger the dockerized code path.